.cloudbuild/scripts/cmd/unit-tests/main.go

```diff
@@ -140,8 +141,7 @@ func run() error {
 		return trace.Wrap(err, "Failed analyzing code")
 	}
 
-	hasOnlyDocChanges := ch.Docs && (!ch.Code)
```

Comment:
I wondered about this, but I couldn't imagine a situation in which this would be problematic. I'd definitely appreciate more feedback on this part though.

Reply:
I expect I was just trying to be emphatic about it being a docs-only change. But I think your simpler logic (i.e. if any code changes, then run tests) is better.